src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -582,19 +593,25 @@ export class ExtensionEditor extends BaseEditor {
 		return Promise.resolve(null);
 	}
 
-	private async openMarkdown(cacheResult: CacheResult<string>, noContentCopy: string, template: IExtensionEditorTemplate): Promise<IActiveElement> {
+	private async openMarkdown(cacheResult: CacheResult<string>, noContentCopy: string, template: IExtensionEditorTemplate, webviewIndex: number): Promise<IActiveElement> {
```

Comment:
`webviewIndex` should probably be an enum instead so that the code is more legible

Reply:
Good point. Done.